doc/user_guide/user_guide.md

```diff
@@ -8,6 +8,10 @@ Syntax definitions in this document are written in [Augmented Backus-Naur Form (
 
 ## Getting Test Containers Into Your Project
 
+Please check out ["Introduction to the Dependency Mechanism"](http://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html), if you want to learn about how maven handles dependencies and dependency scopes.
+
+We assume here that you are familiar with the basics.
+
```

Comment:
theses block would fit better after line: "Exasol test containers are built using Apache Maven, so integrating the release packages into your project is easy with Maven."

Reply:
You are right. I moved too few lines up apparently.
Fixed.